lib/archethic/transaction_chain.ex

```diff
@@ -506,6 +510,41 @@ defmodule Archethic.TransactionChain do
     |> Enum.map(fn {:ok, res} -> res end)
   end
 
+  @doc """
+  Retrieve the genesis address of a chain
+  """
+  @spec resolve_last_address(binary()) :: {:ok, binary()} | {:error, :network_issue}
```

Comment:
Spec definition so not martch the function name.
Same for `fetch_genesis_address_remotely`